sdk/keyvault/Azure.Security.KeyVault.Administration/tests/BackupRestoreTestBase.cs

```diff
@@ -11,6 +11,7 @@
 
 namespace Azure.Security.KeyVault.Administration.Tests
 {
+    [NonParallelizable]
     [IgnoreServiceError(404, "NotFound", Message = "The given jobId is not found", Reason = "Backup/restore tests have inherent concurrency issues")]
     [IgnoreServiceError(409, "Conflict", Message = "User triggered Restore operation is in progress", Reason = "Backup/restore tests have inherent concurrency issues")]
     public abstract class BackupRestoreTestBase : AdministrationTestBase
```

Comment:
This shouldn't have any impact for the reasons you mentioned in the description.

Reply:
Likely not, but it does also make it more local in case something up the chain of base classes changes (including on my own). This is more of a correctness change. Yes, I doubt it'll actually fix the issue. Think of it more as "future proofing".

Reply:
Yeah, but it introduces ambiguity - others may be unsure whether they need a similar change.

Reply:
If they want to make sure no future changes to base classes in or out of their control don't suddenly parallelize, why shouldn't they?

Reply:
I think the issue is that people looking at the code won't realize that this line does nothing. If they remove it and test they would expect it to run in parallel, but it still wouldn't because of the base class. I think a comment would at least help clear this up.

Reply:
I'll add a comment. But there's always `git blame` where I also described it. ;)

Reply:
Thank for your humoring me.